Webpack and Babel Task-1/My-App/src/App.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+import Veg from './components/Veg';
+import Nonveg from './components/Nonveg';
+import Smoothies from './components/Smoothies';
+
+import React from 'react'
+
+export default function App() {
+    const [state, setState] = useState("");
```

Comment:
Try not using state and setState as variable name, use meaningful name

Reply:
> Try not using state and setState as variable name, use meaningful name

done